code/modules/clothing/glasses_vr.dm

```diff
@@ -0,0 +1,58 @@
+/obj/item/clothing/glasses/proc/prescribe(var/mob/user)
+	prescription = !prescription
+
+	//Look it's really not that fancy. It's not ACTUALLY unique scrip data.
+	if(prescription)
+		name = "[initial(name)] (pr)"
+		user.visible_message("[user] replaces the lenses in \the [src] with a new prescription.")
+	else
+		name = "[initial(name)]"
+		user.visible_message("[user] replaces the prescription lenses in \the [src] with generics.")
+
+	playsound(user,'sound/items/screwdriver.ogg', 50, 1)
```

Comment:
This can and probably should go into the base glasses file.

Reply:
I originally attempted to move it into the glasses file to start with, ended up with errors, so I kept it separated.

Reply:
If you can get me the errors I can can probably walk you through how to fix it.